### PR TITLE
Merge management into main

### PR DESCRIPTION
**Pull request by MistzSoftware**
This pull request includes significant updates to documentation, including the addition of a Code of Conduct, Contributing Guidelines, and detailed weekly reports. These changes aim to establish clear community standards, provide guidance for contributors, and track project progress.

Documentation Added:

* [`CODE_OF_CONDUCT.md`](diffhunk://#diff-ffdbe3a1e7ee93cacfc080b6c635ccf3a8f6b0f00f2fb884f78c6b5f9dac8fd2R1-R128): Added a comprehensive Contributor Covenant Code of Conduct to ensure a harassment-free experience for all community members.
* [`CONTRIBUTING.md`](diffhunk://#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R1-R52): Introduced Contributing Guidelines, including instructions for forking and cloning the repository, branching strategy, code style, commit messages, and pull request procedures.

Project management updates:

Updating the readme to include the new member of the team @Loic-nogues .
Adding the weekly reports of weeks 1 and 2
Filling the management artifact

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
```
[the remark this answers is not in the document]
Comment (MistzSoftware) [reply CODE_OF_CONDUCT.md:23]: Fixed

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
```
[the remark this answers is not in the document]
Comment (MistzSoftware) [reply CODE_OF_CONDUCT.md:32]: Fixed

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+* Public or private harassment
```
[the remark this answers is not in the document]
Comment (MistzSoftware) [reply CODE_OF_CONDUCT.md:33]: Fixed

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+* Publishing others' private information, such as a physical or email
```
[the remark this answers is not in the document]
Comment (MistzSoftware) [reply CODE_OF_CONDUCT.md:34]: Fixed

**Review by Loic-nogues — APPROVED** [review]
Perfect

**Review by Furimizu — APPROVED** [review]

**Pull request merged**